packages/web3-core-method/src/index.js

```diff
@@ -706,12 +706,13 @@ Method.prototype.buildCall = function () {
                 // ETH_SENDTRANSACTION
                 if (payload.method === 'eth_sendTransaction') {
                     var tx = payload.params[0];
-                    wallet = getWallet((_.isObject(tx)) ? tx.from : null, method.accounts);
+                    wallet = getWallet((typeof tx === 'object' && !!tx) ? tx.from : null, method.accounts);
 
 
                     // If wallet was found, sign tx, and send using sendRawTransaction
                     if (wallet && wallet.privateKey) {
-                        var txOptions = _.omit(tx, 'from');
+                        var txOptions = JSON.parse(JSON.stringify(tx));
```

Comment:
```suggestion
```

`_.omit` [takes an object](https://underscorejs.org/#omit), so we can assume `tx` is an object